ironfish/src/network/peerNetwork.ts

```diff
@@ -1361,13 +1358,14 @@ export class PeerNetwork {
       return
     }
 
-    if (this.node.memPool.acceptTransaction(transaction)) {
-      this.onTransactionAccepted.emit(transaction, received)
-    }
+    this.onTransactionAccepted.emit(transaction, received)
```

Comment:
Not checking the result of `memPool.acceptTransaction` anymore because that was incorrect. Even if it is not accepted for some reason like it's expired or it has overlapping nullifiers with existing transactions it still should be sent to telemetry as a new transaction seen.

Reply:
What do you think about renaming to `onTransactionSeen`? The event is only used for the telemetry event anyway, and now it's emitted when a new transaction is seen rather than accepted, I suppose

Reply:
Yea I originally had that exact rename in this PR but removed it so it was easier to review with less changes 😆

Reply:
I can add it back in after you finish reviewing